sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClientTest.java

```diff
@@ -587,6 +650,23 @@ private static boolean createBodyContentEquals(Object requestBody, CreateQueueOp
             && "application/xml".equals(content.getType());
     }
 
+    private static boolean verifyAdditionalAuthHeaderPresent(Context context, String requiredHeader, String token) {
+        return context.getData(AZURE_REQUEST_HTTP_HEADERS_KEY).map(headers -> {
+            if (headers instanceof HttpHeaders) {
```

Comment:
the nested if statements make it hard to read. I'd consider returning earlier if something doesn't match.

```java
if (!(headers instanceof HttpHeaders)) {
}
```

Reply:
done